replication/client/client.go

```diff
@@ -45,7 +47,7 @@ var (
 
 func init() {
 	logger.SetOutput(os.Stdout)
-	logger.SetLevel(logrus.InfoLevel)
+	logger.SetLevel(logrus.DebugLevel)
```

Comment:
On purpose?

Reply:
Yeah just for now while I debug things. Will remove before merging.